### PR TITLE
Fix parsing exception for string list properties

### PR DESCRIPTION
**Pull request by joeybrk372**
- [x] joeferner
- [ ] mwizeman joeybrk372 jharwig sfeng88

Testing Instructions:

Points of Regression: Adding/removing/collapsing of entities on the graph

CHANGELOG
Fixed: When performing an undo of collapsed entities on a graph product, the product would become blank permanently.

### CODE REVIEW

File: core/core/src/main/java/org/visallo/core/model/properties/types/StringListSingleValueVisalloProperty.java [hunk 1]
```diff
@@ -16,10 +16,19 @@ public String wrap(List<String> value) {
...
     public List<String> unwrap(Object value) {
         if (value == null) {
             return null;
         }
-        return JSONUtil.toStringList(JSONUtil.parseArray(value.toString()));
+
+        List<String> valueList;
+        if (value instanceof String) {
+            valueList = JSONUtil.toStringList(JSONUtil.parseArray(value.toString()));
+        } else {
```
Comment (joeferner) [inline core/core/src/main/java/org/visallo/core/model/properties/types/StringListSingleValueVisalloProperty.java]: can you change this to `else if (value instanceof List) { ... } else { throw new VisalloException("Could not unwrap type: " + value.getClass().getName()); }

**Comment by sfeng88** [comment]
@joeferner can you re-review this PR when you get a chance please?

**Comment by sfeng88** [comment]
@jharwig looks like there are some test failures in `visallo-web-test`

**Comment by sfeng88** [comment]
@joeferner is this ready to be tested?

**Comment by joeferner** [comment]
@sfeng88 good to go

**Comment by sfeng88** [comment]
LGTM

**Pull request merged**